edenia/antelope-tools
# Perf/improve ath calculation performance

### Improve Response Time

### What does this PR do?

- Resolve #1395 
- Solve the problem related to hasura response time in jungle evm
- Get the second with the maximun transactions from the last 30 minutes instead of all blocks from the DB to update the ATH
- Get the second with the maximun transactions from the current old blocks to update the ATH
- Increase the timeout when the synchronization of old blocks ends
 
### Steps to test

1. Run the project locally with evm configuration
2. Check the ATH calculation
